src/Popup.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prop-types */
+import './style.css';
+import React, { useState } from 'react'
+import Memo from './Memo'
+
+function Popup({ setValC, events, element, month, year }) {
+    const [ , addContent ] = useState(0);
+    const[ memo, memoClicked ] = useState(false);
+    const x = element.getBoundingClientRect().right;
+    const y = element.getBoundingClientRect().top - 5;
+    const arrowStyle = {
+        left :`${ x }px`,
+        top : `${ y + 35 }px`
+    }
+    const rectStyle = {
+        left :`${ x + 20 }px`,
+        top : `${ y - 5 }px`
+    }
+    const index = `${ element.innerText }/${ month }/${ year }`;
+    const eventArray = events[ index ];
```

Comment:
events as an entity is having 2 source of truth, one at App and the other here. Merge them into one.